src/components/About/NitsHacks/NitsHacks.module.scss

```diff
@@ -0,0 +1,226 @@
+@import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono&display=swap');
+
+@font-face {
+    font-family: Pixeloid;
+    src: url("../../../../public/fonts/PixeloidSans-Bold.ttf");
```

Comment:
change to `/fonts/[path]`. files in public directory are served at the root path.